packaging/installer/netdata-updater.sh

```diff
@@ -73,14 +73,45 @@ download() {
   fi
 }
 
-set_tarball_urls() {
-  local extension="tar.gz"
+function parse_version() {
+  r="${1}"
+  if echo "${r}" | grep -q '^v.*'; then
```

Comment:
nit: you could just remove the starting v uncondtionally (ie `sed 's/^v//'`) instead of this if block